cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/loadbalancer/AwsLoadBalancerScheme.java

```diff
@@ -1,16 +1,23 @@
 package com.sequenceiq.cloudbreak.cloud.aws.loadbalancer;
 
 public enum AwsLoadBalancerScheme {
-    PUBLIC("internet-facing"),
-    PRIVATE("internal");
+    INTERNET_FACING("internet-facing", "External"),
```

Comment:
This resource isn't persisted. I decided to do the name refactor to match what's actually used by AWS, which makes things on the AWS side more clear. The reason it was PUBLIC/PRIVATE before is so that I could convert directly between this and `LoadBalanceType`, but in https://github.com/hortonworks/cloudbreak/pull/9693 I added a converter for those enums so copying the names directly is no longer required.